src/main/java/com/sprint/mission/discodeit/dto/authDto/LoginDto.java

```diff
@@ -0,0 +1,4 @@
+package com.sprint.mission.discodeit.dto.authDto;
```

Comment:
[구글 Java 스타일 가이드](https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names)에 따르면 패키지명은 소문자와 숫자만을 사용해요.   

> Package names use **only lowercase letters and digits** (no underscores). Consecutive words are simply concatenated together. For example, com.example.deepspace, not com.example.deepSpace or com.example.deep_space.  

```suggestion
package com.sprint.mission.discodeit.dto.authdto;
```